### PR TITLE
Fix bug when form reply event in allOf list

### PR DESCRIPTION
**Pull request by yinan-symphony**
Form reply event in a all-of list is construct as a message
element and it must be exclusive without having a timeout constrait,
otherwise it does not make sense in this particular use case.

### Description
Please put here the intent of your pull request.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [ ] Referenced a ticket in the PR title or description
- [ ] Filled properly the description and dependencies, if any
- [ ] Unit/Integration tests updated or added
- [ ] Javadoc added or updated
- [ ] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/builder/FormRepliedNodeBuilder.java [hunk 1]
```diff
@@ -29,15 +29,10 @@ public class FormRepliedNodeBuilder extends AbstractNodeBpmnBuilder {
   public AbstractFlowNodeBuilder<?, ?> build(WorkflowNode element, String parentId,
       AbstractFlowNodeBuilder<?, ?> builder, BuildProcessContext context) {
     if (builder instanceof ParallelGatewayBuilder) {
-      return builder.exclusiveGateway()
-          .intermediateCatchEvent()
-          .timerWithDuration(readTimeout(element))
```
Comment (symphony-soufiane) [inline workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/builder/FormRepliedNodeBuilder.java:34]: we don't put timeout duration ?

File: workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/SendMessageExecutor.java [hunk 2]
```diff
@@ -130,7 +130,13 @@ private List<String> resolveStreamId(ActivityExecutorContext<SendMessage> execut
       V4UserJoinedRoom event = (V4UserJoinedRoom) execution.getEvent().getSource();
       return singletonList(event.getStream().getStreamId());
 
-    } else {
+    } else if (execution.getEvent() != null
+        && execution.getEvent().getSource() instanceof V4SymphonyElementsAction) {
+      // or retrieved from the current even
+      V4SymphonyElementsAction event = (V4SymphonyElementsAction) execution.getEvent().getSource();
```
Comment (symphony-soufiane) [inline workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/SendMessageExecutor.java:136]: the return can be done in one line I guess

File: workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/SendMessageExecutor.java [hunk 3]
```diff
@@ -130,7 +130,13 @@ private List<String> resolveStreamId(ActivityExecutorContext<SendMessage> execut
...
+      return singletonList(event.getStream().getStreamId());
+
+    }else {
```
Comment (symphony-soufiane) [inline workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/SendMessageExecutor.java:139]: bad formatting here :)

**Review by symphony-soufiane — APPROVED** [review]

File: workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/builder/FormRepliedNodeBuilder.java (hunk 1 above)
Comment (yinan-symphony) [reply workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/builder/FormRepliedNodeBuilder.java:34]: we can have a short discuss on it. i think it is not necessary

**Pull request merged**